ResearchKit/ActiveTasks/ORKAudioStepViewController.h

```diff
@@ -37,6 +37,8 @@ NS_ASSUME_NONNULL_BEGIN
 ORK_CLASS_AVAILABLE
 @interface ORKAudioStepViewController : ORKActiveStepViewController
 
+@property (nonatomic, assign) CGFloat alertThreshold;
```

Comment:
Is a negative value a valid value? Should we use the negative value to restore the default value?